tests/backup/backup_helper.go

```diff
@@ -1490,3 +1493,54 @@ func IsPresent(dataSlice interface{}, data interface{}) bool {
 	return false
 
 }
+
+// return current stork image.
+func getStorkImage() (string, error) {
+	storkDeployment, err := apps.Instance().GetDeployment(storkDeploymentName, storkDeploymentNamespace)
+	if err != nil {
+		return "", err
+	}
+	storkImage := storkDeployment.Spec.Template.Spec.Containers[0].Image
+	return storkImage, nil
+}
+
+// Upgrade the stork to required version.
+func upgradeStorkVersion(reqStorkImage string) (v1.ConditionStatus, error) {
+
+	currentStorkImage, _ := getStorkImage()
+	currentStorkVersionStr := strings.Split(currentStorkImage, ":")[len(strings.Split(currentStorkImage, ":"))-1]
```

Comment:
Instead of doing this, you modify the function `getStorkImage`, rename it to `getStorkVersion` and you should get only the version like `2.12.1` or `23.2`

Reply:
Done, now the getStorkImageVersion will return stork image version